sdk/src/main/java/com/kin/ecosystem/web/EcosystemWebViewClient.java

```diff
@@ -24,4 +31,29 @@ public void onPageFinished(WebView view, String url) {
     public void onReceivedError(WebView view, WebResourceRequest request, WebResourceError error) {
 		Logger.log(new Log().withTag(TAG).text("onReceivedError(WEB_VIEW, REQUEST, \"" + error.toString() + "\")"));
     }
+
+    @Override
+    public boolean shouldOverrideUrlLoading(WebView view, String url) {
+        if (url.startsWith("mailto:")) {
+            MailTo mt = MailTo.parse(url);
+            Intent intent = newEmailIntent(context, mt.getTo(), mt.getSubject(), mt.getBody(), mt.getCc());
+            context.startActivity(intent);
+
+            return true;
+        } else {
+            view.loadUrl(url);
```

Comment:
?

Reply:
"Note: Do not call WebView.loadUrl(String) with the same URL and then return true. This unnecessarily cancels the current load and starts a new load with the same URL. The correct way to continue loading a given URL is to simply return false, without calling WebView.loadUrl(String)."